content-by-language/java/outbound-integration.java

```diff
@@ -10,55 +12,56 @@
 import org.apache.http.impl.client.CloseableHttpClient;
 import org.apache.http.impl.client.HttpClientBuilder;
 import org.apache.http.ssl.SSLContextBuilder;
-import java.io.FileInputStream;
+
 import java.io.IOException;
 import java.nio.charset.StandardCharsets;
 import java.security.GeneralSecurityException;
 import java.security.KeyStore;
 import java.security.cert.Certificate;
 import java.security.cert.CertificateFactory;
-import javax.net.ssl.SSLContext;
 
-public class App {
-public static void main(String[] args) throws IOException, GeneralSecurityException {
+import javax.net.ssl.SSLContext;
 
-  final String proxyHost = "{VAULT_HOST}";
-  final int proxyPort = {PORT};
-  final String proxyUser = {USERNAME};
-  final String proxyPassword = {PASSWORD};
-  HttpHost proxy = new HttpHost(proxyHost, proxyPort);
+public class OutboundIntegration {
+  public static void main(String[] args) throws IOException, GeneralSecurityException {
+    final String proxyHost = "{VAULT_URL}";
+    final int proxyPort = {PORT};
+    final String proxyUser = "{USERNAME}";
+    final String proxyPassword = "{PASSWORD}";
+    final HttpHost proxy = new HttpHost(proxyHost, proxyPort);
 
-  CredentialsProvider provider = new BasicCredentialsProvider();
-  provider.setCredentials(AuthScope.ANY,
-                          new UsernamePasswordCredentials(proxyUser, proxyPassword));
+    final CredentialsProvider provider = new BasicCredentialsProvider();
+    provider.setCredentials(AuthScope.ANY,
+        new UsernamePasswordCredentials(proxyUser, proxyPassword));
 
-  CloseableHttpClient client = HttpClientBuilder
-    .create()
-    .setProxy(proxy)
-    .setDefaultCredentialsProvider(provider)
-    .setSSLContext(buildSSLContext())
-    .build();
+    final CloseableHttpClient client = HttpClientBuilder
+        .create()
+        .setProxy(proxy)
+        .setDefaultCredentialsProvider(provider)
+        .setSSLContext(buildSSLContext())
+        .build();
 
-  HttpPost httpPost = new HttpPost("{VGS_SAMPLE_ECHO_SERVER}/post");
-  httpPost.setHeader("Content-Type", "application/json");
-  httpPost.setEntity(new StringEntity("{\"account_number\":\"{ALIAS}\"}"));
+    final HttpPost httpPost = new HttpPost("{VGS_SAMPLE_ECHO_SERVER}/post");
+    httpPost.setHeader("Content-Type", "application/json");
+    httpPost.setEntity(new StringEntity("{\"account_number\":\"tok_sandbox_oTktJmBaAhUyGHnNJt7WMA\"}"));
```

Comment:
should it be hardcoded here? I can add it at personalization step, so:
- for test user we would have this alias
- for logged in user would be `{ALIAS}` but with tooltip that here should be some redacted data